lib/fast-path.js

```diff
@@ -99,12 +83,12 @@ FPp.valueIsNode = function () {
 };
 
 FPp.getNode = function (count) {
-  if (count === undefined) count = 0;
+  count = typeof count === "undefined" ? 0 : count;
```

Comment:
I noticed in other places you use `typeof` for `"undefined"` checks so this is just for consistency. Though it could still be an `if` block if you want.